templates/common/_base/files/cleanup-cni-conf.yaml

```diff
@@ -7,3 +7,4 @@ contents:
     r /etc/kubernetes/cni/net.d/10-ovn-kubernetes.conf
     r /etc/kubernetes/cni/net.d/00-multus.conf
     d /run/multus/cni/net.d/ 0755 root root - -
+    D /var/lib/cni/networks/openshift-sdn/ 0755 root root - -
```

Comment:
should this 'd' ? (i dont know just asking)

Reply:
slack discussion... we want the 'D' to remove the tree.

Reply:
No, we want to remove the directory too.  If we ever support migration of a cluster from one plugin to another, then it makes sense to get rid of the directory entirely.